src/runtime/hexagon/fastrpc/tvm_hexagon_remote.h

```diff
@@ -0,0 +1,138 @@
+/*!
+ * Copyright 2019 Qualcomm Innovation Center, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE‐2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * This Contribution is being provided by Qualcomm Technologies, Inc.,
+ * a Delaware corporation, or its subsidiary Qualcomm Innovation Center, Inc.,
+ * a California corporation, under certain additional terms and conditions
+ * pursuant to Section 5 of the Apache 2.0 license.  In this regard, with
+ * respect to this Contribution, the term "Work" in Section 1 of the
+ * Apache 2.0 license means only the specific subdirectory within the TVM repo
+ * (currently at https://github.com/dmlc/tvm) to which this Contribution is
+ * made.
+ * In any case, this submission is "Not a Contribution" with respect to its
+ * permitted use with any of the "vta" and "verilog" subdirectories in the TVM
+ * repo.
+ */
+#ifndef TVM_RUNTIME_HEXAGON_FASTRPC_TVM_HEXAGON_REMOTE_H_
+#define TVM_RUNTIME_HEXAGON_FASTRPC_TVM_HEXAGON_REMOTE_H_
+/// @file tvm_hexagon_remote.idl
+///
+// qidl copyright
+// qidl nested=false
+#include "AEEStdDef.h"
+#include "android_Release_aarch64/remote.h"
```

Comment:
Do we have preference to use `<>` instead of quotes?

Reply:
Is that a question for me?  I don't know.  You tell me...